pkg/component/operator/audio/v0/README.mdx

```diff
@@ -5,10 +5,10 @@ draft: false
 description: "Learn about how to set up a VDP Audio component https://github.com/instill-ai/instill-core"
 ---
 
-The Audio component is an operator component that allows users to extract and manipulate audio from different sources.
+The Audio component is an operator component that allows users to operate audio data..
```

Comment:
```suggestion
The Audio component is an operator component that allows users to operate audio data.
```

Reply:
The double periods have to do with the new description rule (i.e., always use period) resulting the doc generated repeating the period due to the template hasn't been updated (another PR will do this). But yes, this reminds me that we can actually just manually change this actually.